hack/update-codegen.sh

```diff
@@ -5,6 +5,8 @@ set -e
 DIR=$(dirname "$0")
 cd "$DIR/.."
 
+goversion=$(grep "^go " go.mod | sed 's/go //')
```

Comment:
Ooo this is clever - I wish there was a `go mod` command to query this directly!